package.json

```diff
@@ -19,23 +19,24 @@
   ],
   "scripts": {
     "backupApis": "rm -rf updateApisTmp && mkdir updateApisTmp && mv apis updateApisTmp/oldApis",
-    "build": "npm run clean && npm run renderTemplates && tsc && eslint dist --ext .ts,.js --quiet --fix",
+    "build": "npm run clean && npm run renderTemplates && npm run compile",
+    "ci": "rm -rf node_modules && yarn install",
     "clean": "rm -rf renderedTemplates dist",
-    "depcheck": "depcheck",
-    "diffApis": "raml-toolkit diff --dir updateApisTmp/oldApis apis -f console -o updateApisTmp/ramlDiff.txt",
+    "compile": "tsc && eslint dist --ext .ts,.js --quiet --fix",
+    "diffApis": "raml-toolkit diff --dir updateApisTmp/oldApis apis -f console -o updateApisTmp/oasDiff.txt -s oas",
     "doc": "npm run doc:generate && npm run doc:resources",
     "doc:generate": "typedoc --mode modules --out ./docs renderedTemplates/** --tsconfig tsconfig.json --external-modulemap  \".*/renderedTemplates/([\\w\\-_]+)/\"",
     "doc:resources": "cp CHANGELOG.md ./docs",
     "lint": "npm run lint:dev -- --quiet",
     "lint:dev": "sort-package-json && eslint . --ext .ts --fix",
     "prepack": "npm run build",
     "prepare": "snyk protect",
-    "renderTemplates": "ts-node src/generate.ts",
+    "renderTemplates": "ts-node src/generate-oas.ts",
     "snyk:auth": "snyk auth",
-    "pretest": "npm run lint && npm run depcheck",
```

Comment:
Removing depcheck since it seems to think the devDependencies we are using for tests are not used

ie:
```
yarn run v1.22.22
$ depcheck
Unused devDependencies
* @types/chai
* @types/fs-extra
* @types/handlebars-helpers
* @types/node-fetch
* @types/sinon
* chai
* fs-extra
* handlebars-helpers
* nock
* sinon
error Command failed with exit code 255.
```

Reply:
I was able to readd the depcheck once I switched to using `npm` instead of `yarn`